CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.1.4 - Not yet released
```

Comment:
Maybe also bump the version in the pubspec.yaml file?

Reply:
Done.